.core_files.yaml

```diff
@@ -0,0 +1,119 @@
+# Defines a list of files that are part of main core of Home Assistant.
+# Changes to these files/filters define how our CI test suite is ran.
+core: &core
+  - homeassistant/*.py
+  - homeassistant/auth/**
+  - homeassistant/helpers/*
+  - homeassistant/package_constraints.txt
+  - homeassistant/util/*
+  - pyproject.yaml
+  - requirements.txt
+  - setup.cfg
+
+# Our base platforms, that are used by other integrations
+base_platforms: &base_platforms
+  - homeassistant/components/air_quality/*
+  - homeassistant/components/alarm_control_panel/*
+  - homeassistant/components/binary_sensor/*
+  - homeassistant/components/button/*
+  - homeassistant/components/calendar/*
+  - homeassistant/components/camera/*
+  - homeassistant/components/climate/*
+  - homeassistant/components/cover/*
+  - homeassistant/components/device_tracker/*
+  - homeassistant/components/fan/*
```

Comment:
```suggestion
  - homeassistant/components/fan/*
  - homeassistant/components/geo_location/*
```

Reply:
LOL I've added it to hassfest in a local branch, but not to this file 🤦 

Thanks!